lib/presentation/pages/order/widgets/order_cart.dart

```diff
@@ -0,0 +1,164 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:furniture_shop/presentation/widgets/base/custom_text.dart';
+import 'package:furniture_shop/values/colors.dart';
+import 'package:furniture_shop/values/dimens.dart';
+import 'package:furniture_shop/values/font_sizes.dart';
+
+class OrderCard extends StatelessWidget {
```

Comment:
Add Button into OrderCard (to navigate to order detail)

Reply:
done